pandas/io/excel/_base.py

```diff
@@ -781,14 +778,27 @@ def close(self):
         return self.save()
 
 
+def _is_ods_stream(stream):
+    stream.seek(0)
+    is_ods = False
+    if stream.read(4) == b"PK\003\004":
+        stream.seek(30)
+        is_ods = (
+            stream.read(54) == b"mimetype"
+            b"application/vnd.oasis.opendocument.spreadsheet"
+        )
+    stream.seek(0)
+    return is_ods
+
+
 class ExcelFile:
     """
     Class for parsing tabular excel sheets into DataFrame objects.
     Uses xlrd. See read_excel for more documentation
 
     Parameters
     ----------
-    io : str, path object (pathlib.Path or py._path.local.LocalPath),
+    path_or_io : str, path object (pathlib.Path or py._path.local.LocalPath),
```

Comment:
path

Reply:
This can be a path or a file-like object. The current doc-string is wrong.

Reply:
kk, rename to path_or_buffer

Reply:
Changed